data/account/participationRegistry_test.go

```diff
@@ -17,28 +17,24 @@
 package account
 
 import (
-	"context"
-	"database/sql"
-	"fmt"
-	"github.com/algorand/go-algorand/data/basics"
-	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
+	"github.com/algorand/go-algorand/data/basics"
 	"github.com/algorand/go-algorand/test/partitiontest"
 	"github.com/algorand/go-algorand/util/db"
 )
 
-func getRegistry(t *testing.T) ParticipationRegistry {
+func getRegistry(t *testing.T) *participationDB {
 	rootDB, err := db.OpenPair(t.Name(), true)
 	require.NoError(t, err)
 
 	registry, err := MakeParticipationRegistry(rootDB)
 	require.NoError(t, err)
 	require.NotNil(t, registry)
 
-	return registry
+	return registry.(*participationDB)
```

Comment:
you shouldn't need to do that. instead, create a `makeParticipationRegistry` that would return a `* participationDB`, and use it internally. in the `MakeParticipationRegistry`, just return the external facing interface.